### PR TITLE
fixes #1

### LINKED ISSUE #1: Make it clearer that the config client is a separate application
The guide lacks clear instructions for creating the client app, so "add this dependency" sounds like you need to add it to the config server. The instructions later on need the actuator for the /refresh endpoint as well, and I don't think that was called out anywhere.

### PR DESCRIPTION
**Pull request by joshlong**
changed to reflect @dsyer 's excellent requested updates

### CODE REVIEW

**Comment by dsyer** [comment]
I don't object to this change, but it doesn't fix #1 if you ask me. IMO we need complete, clear instructions about how to create a new app and add the correct dependencies. Did I miss it?

**Comment by spring-operator** [comment]
There's a whole section ('Reading Configuration from the Config Server
using the Config Client') that articulates that we're standing up a
separate app that needs a separate dependency. It also clarifies that the
refresh endpoint requires Actuator. It spells out the files required for
both the configuration-service and the configuration-client and spells out
that they run one after the other. Maybe I'm missing it? I'm happy to take
another swing at it (and of course would welcome your edits) but I'm
struggling to see what you're seeing. To me it reads as how to build two
Boot apps.

On Friday, January 22, 2016, Dave Syer notifications@github.com wrote:

> I don't object to this change, but it doesn't fix #1
> https://github.com/spring-guides/gs-centralized-configuration/issues/1
> if you ask me. IMO we need complete, clear instructions about how to create
> a new app and add the correct dependencies. Did I miss it?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/spring-guides/gs-centralized-configuration/pull/2#issuecomment-173855364
> .

## 

Thanks,
Josh Long
Spring Developer Advocate
Pivotal
http://www.joshlong.com || joshlong.com || http://twitter.com/starbuxman

**Comment by dsyer** [comment]
I think it's probably me. I assume that the reader of the guide can follow instructions from a completely clean slate and get to the final result. These instructions don't support that workflow. Maybe they support the alternative workflow of "take the 'initial' project and modify it"? I think if that's the case it should be more obvious that this is the expected workflow. I also think we should question if that is what we want because all the other guides IIRC take the approach of building from nothing and giving the "initial" state only as a stepping stone.

**Comment by joshlong** [comment]
i'm re-reading it. the build files for both modules are dumped at the very beginning, i've just noticed. by the time you get to the client module, it isn't spelled out that you should scroll up to grab the secondary build file and re-run the directory creation procedure. i think that contributes to the confusion.

File: README.adoc [hunk 1]
```diff
@@ -130,10 +130,10 @@ include::complete/configuration-client/src/main/java/hello/ConfigClientApplicati
...
 
 
 == Summary
-Congratulations! You've just used Spring to centralize configuration for all your services and to then dynamically update configuration.
+Congratulations! You've just used Spring to centralize configuration for all your services by first standing up a  and to then dynamically update configuration.
```
Comment (gregturn) [inline README.adoc:137]: @joshlong Standing up a what?

**Pull request merged**